lib/js/components/TextGroup/TextGroup.vue

```diff
@@ -0,0 +1,288 @@
+<template>
+	<div
+		class="ds-textGroup"
+		:class="{
+			'-ds-small': isSmall,
+
+			'-ds-hovered': state === TEXT_GROUP_STATES.HOVERED,
+			'-ds-loading': isLoading,
+			'-ds-disabled': state === TEXT_GROUP_STATES.DISABLED,
+
+			'-ds-interactive': isInteractive,
+			'-ds-selected': isSelected,
+
+			[loadingSizeClassName]: isLoading,
+
+			[colorClassName]: true,
+		}"
+	>
+		<div
+			v-if="eyebrowText !== null"
+			class="ds-textGroup__eyebrow"
+			:class="{
+				'-ds-ellipsis': eyebrowTextEllipsis,
+				'-ds-uppercase': isEyebrowTextUppercase,
+			}"
+		>
+			<ds-skeleton
+				v-if="isLoading"
+				class="ds-textGroup__loadingSkeleton"
+				width="50%"
+				:height="isSmall ? '8px' : '10px'"
+			/>
+			<span v-else>{{ eyebrowText }}</span>
+		</div>
+		<div
+			v-if="mainText !== null"
+			class="ds-textGroup__main"
+			:class="{
+				'-ds-ellipsis': mainTextEllipsis,
+			}"
+		>
+			<ds-skeleton
+				v-if="isLoading"
+				class="ds-textGroup__loadingSkeleton"
+				width="100%"
+				:height="isSmall ? '14px' : '16px'"
+			/>
+			<span v-else>{{ mainText }}</span>
+		</div>
+		<div
+			v-if="supportingText !== null"
+			class="ds-textGroup__supporting"
+			:class="{
+				'-ds-ellipsis': supportingTextEllipsis,
+			}"
+		>
+			<ds-skeleton
+				v-if="isLoading"
+				class="ds-textGroup__loadingSkeleton"
+				width="100%"
+				height="14px"
+			/>
+			<span v-else>{{ supportingText }}</span>
```

Comment:
In https://github.com/bethinkpl/wnl-platform/pull/5124#discussion_r1706897380 we're going to pass HTML into this prop. How do we handle HTML tags then? Escaped HTML doesn't look ok

Reply:
https://github.com/bethinkpl/design-system/pull/401/commits/a4f7027f06791412b682628f894ba785d4a56cd2